roles/zuul/files/jobs/lore.yaml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
We don't want this initial `---` because zuul will read all files in the directory and concatenate them, but this will trigger a new yaml segment separating this job and builders from previously-defined jobs and builders.